### PR TITLE
fix some broken links reported by link-checker

### PR DESCRIPTION
**Pull request by masinter**

### CODE REVIEW

File: content/en/news/2021MedleyAnnualReport.md [hunk 1]
```diff
@@ -63,27 +63,27 @@ One of the reasons for restoring Medley Interlisp is to support the revival of a
 * **LFG:** the Grammar Writer’s Workbench,
 * **LispUsers:** User-contributed software (before “open source” was a thing). We’ve [checked out over half of the LispUsers files](https://docs.google.com/spreadsheets/d/1pn4UcS-9CgMLi\_qeGZlOGGEusAKsNDKxz1XhLwQCgKw/edit?usp=sharing).
 
-### Documentation <a href="#_ufh34bx5j0eg" id="_ufh34bx5j0eg"></a>
+### Documentation
 
-#### Books <a href="#_5tyqara8iuk4" id="_5tyqara8iuk4"></a>
+#### Books 
 
 * _****_[_**Interlisp: The Language and Its Usage**_](../../documentation/interlisp-books.md), by Steve Kaisler, originally published by John Wiley & Sons, scanned and converted to a compact PDF. This book describes the core features of the Interlisp language.
```
Comment (stumbo) [inline content/en/news/2021MedleyAnnualReport.md]:
```
[Interlisp: The Language and Its Usage](../../documentation/1986-interlisp-language-book-1.pdf)
```

Fixes this issue.  Not sure how everything got so scrambled.

File: content/en/news/2021MedleyAnnualReport.md [hunk 2]
```diff
@@ -63,27 +63,27 @@ One of the reasons for restoring Medley Interlisp is to support the revival of a
...
 * ****[**Medley Interlisp: The Interactive Programming Environmen**t](../../documentation/interlisp-books.md), by Steve Kaisler, describes the Interlisp-D Interactive Programming Environment as implemented on Xerox D-machines and now running on several modern platforms.
```
Comment (stumbo) [inline content/en/news/2021MedleyAnnualReport.md]:
```
[Medley Interlisp: The Interactive Programming Environment](../../documentation/20211225-interlisp-book-2.pdf)
```

**Review by stumbo — CHANGES_REQUESTED** [review]
Fixes for:

> Lines: 389, 390 https://interlisp.org/documentation/interlisp-books.md 404. This was from the wiki or gitbook? There's a interlisp-books.md in the gitbook/wiki translations but it isn't very useful.

added as comments.

**Comment by masinter** [comment]
fixed, thanks

**Review by stumbo — APPROVED** [review]

**Pull request merged**